scripts/xtensa-build-all.sh

```diff
@@ -26,7 +26,7 @@ then
 	echo "       [-k] Use private key"
 	echo "       [-v] Verbose Makefile log"
 	echo "       [-j [n]] Set number of make build jobs." \
-		"Jobs=#cores when no flag. Inifinte when no arg."
+		"Jobs=#cores when no flag. No limit when no arg."
```

Comment:
I think this should be in a separate commit as it doesn't relate to the commit description.

Reply:
I'll drop this typo fix, I don't think it's bad enough to worth a commit alone.

Reply:
Maybe good for the next cleanup patch.

Reply:
Done. Funny enough the checkpatch git commit hook tried to hard to stop me from amending this commit and mispelling Inifinte again :-)